tests/integration/install_from_local_repository/server.yml

```diff
@@ -0,0 +1,24 @@
+---
+- hosts: server
+  become: yes
+  become_method: sudo
+  environment:
+    http_proxy: "{{ http_proxy | default() }}"
```

Comment:
you might not be sure but it does. I only use `default()` when the variable can be undefined. in this case, even if `http_proxy` is undefined, it does not matter. but when a variable SHOULD be defined, i do NOT use `default`, rather prefer to bail out.